examples/model_selection/plot_grid_search_digits.py

```diff
@@ -13,62 +13,149 @@
 
 More details on tools available for model selection can be found in the
 sections on :ref:`cross_validation` and :ref:`grid_search`.
-
 """
 
+print(__doc__)
```

Comment:
I think that we don't include anymore these `print`.

```suggestion
```

Reply:
I confirm, they were useful in the beginning of the project because the sphinx-gallery rendering was primitive but nowadays this is redundant with the content of the HTML rendering.